node/clients/paymentProvider.ts

```diff
@@ -0,0 +1,29 @@
+import { ExternalClient, InstanceOptions, IOContext } from '@vtex/api'
+import { replace } from 'ramda'
+import { PaymentAuthorizationResponse } from '../types'
+import { withProxyAuthorization } from './utils'
+
```

Comment:
Since we're writing an example, I think would be nice add a quick explanation about what this client do @marcosvcp

Reply:
Same for other files that you judge as important for this service